setup.py

```diff
@@ -178,7 +178,12 @@ def detect_libb2(prefixes):
 
 
 with open('README.rst', 'r') as fd:
-    long_description = fd.read()
+    # remove badges
+    long_description = re.compile(r'^.. start-badges.*^.. end-badges', re.M | re.S).sub('', fd.read())
```

Comment:
keep long_description = fd.read() on its own line, to have all the regex stuff following the same pattern and make it easier to add/delete such regex ops?